### PR TITLE
EWL 4634: SG2 | Create default/"Welcome" pattern

### PR DESCRIPTION
**Pull request by JaeBea**
## Ticket(s)

**Jira Ticket**
- [EWL 4634] SG2 | Create default/"Welcome" pattern](https://issues.ama-assn.org/browse/EWL-4634)

## Description

AMA-style-guide, https://americanmedicalassociation.github.io/AMA-style-guide/,  has a welcome page presented when first navigating to the style guide. The welcome page presents some basic information to help guide new users. This feature should also exist for ama-style-guide-2, https://americanmedicalassociation.github.io/ama-style-guide-2/. Currently on ama-style-guide-2 a user is greeted with the default "all" template provided with PatternLab.

This update adds a welcome landing page for ama-style-guide-2 with similar level of guidance as provided in AMA-style-guide.

## To Test
- [ ] Pull `develop` branch.
- [ ] Run `gulp serve` to load style guide landing page
- [ ] Compare with page located at http://localhost:3000/?p=all. These should look the same
- [ ] Pull `feature/EWL-4634-default-welcome-pattern` branch.
- [ ] Run `gulp serve` to load style guide landing page
- [ ] Observe new landing page with AMA logo, headings, and text.

## Visual Regressions

Only 12 fails occur when running `backstop test`. These failures are from a difference in exact date specified on "membership" molecules and broken video image shown for "article stub as video" molecule.

## Relevant Screenshots/GIFs
**Old landing page**
style-guide-landing-page--old

**New landing page**
style-guide-landing-page--new

## Remaining Tasks

N/A

## Additional Notes

- The molecule "article-stub-as-video" shows a broken video

---

### CODE REVIEW

File: styleguide/source/_patterns/05-pages/style-guide-welcome.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    },
+    {
+      "type": "heading",
+      "template": "@atoms/heading.twig",
+      "content": {
+        "level": "2",
+        "text": "Working with our style guide.",
+        "class": "ama__h2"
+      }
+    },{
```
Comment (designerneil) [inline styleguide/source/_patterns/05-pages/style-guide-welcome.json]: need a line break here

File: styleguide/source/_patterns/05-pages/style-guide-welcome.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      "type": "heading",
+      "template": "@atoms/heading.twig",
+      "content": {
+        "level": "3",
+        "text": "Structure",
+        "class": "ama__h3"
+      }
+    },{
```
Comment (designerneil) [inline styleguide/source/_patterns/05-pages/style-guide-welcome.json]: line break

File: styleguide/source/_patterns/05-pages/style-guide-welcome.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      "type": "paragraph",
+      "template": "@atoms/paragraph.twig",
+      "content": {
+        "text": "Each pattern should have its own accompanying <a target=\"_blank\" href=\"https://daringfireball.net/projects/markdown/\">Markdown</a> file that contains a useful description of the pattern and links to the relevant Jira tickets/Github issues relating to that pattern's creation. This information is needed to provide context as to how and when a particular pattern should be used."
+      }
+    }
+  ]
+}
```
Comment (designerneil) [inline styleguide/source/_patterns/05-pages/style-guide-welcome.json]: need extra line

File: styleguide/source/_patterns/05-pages/style-guide-welcome.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        "level": "3",
+        "text": "Structure",
+        "class": "ama__h3"
+      }
+    },{
+      "type": "paragraph",
+      "template": "@atoms/paragraph.twig",
+      "content": {
+        "text": "This living style guide is built with <a target=\"_blank\" href=\"http://patternlab.io/\">Pattern Lab</a>  and adheres to the principles of <a target=\"_blank\" href=\"http://bradfrost.com/blog/post/atomic-web-design/\">Atomic Design</a>"
+
```
Comment (designerneil) [inline styleguide/source/_patterns/05-pages/style-guide-welcome.json]: remove line

File: styleguide/source/_patterns/05-pages/style-guide-welcome.json (hunk 1 above)
Comment (designerneil) [inline styleguide/source/_patterns/05-pages/style-guide-welcome.json]: In general whenever you have a comma put in a line break
 },
{

**Comment by designerneil** [comment]
@JaeBea please use 2 spaces instead of 4

File: styleguide/source/_patterns/05-pages/style-guide-welcome.twig [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{% include '@atoms/site-logo.twig' %}
+
```
Comment (designerneil) [inline styleguide/source/_patterns/05-pages/style-guide-welcome.twig]: remove line

**Comment by designerneil** [comment]
@JaeBea loved your implementation of the welcome page!

**Comment by JaeBea** [comment]
@designerneil Your comments have been integrated.

**Review by labbydev — APPROVED** [review]

**Pull request merged**